recipes/zppy-interfaces/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "zppy-interfaces" %}
+{% set version = "0.1.1" %}
+{% set python_min = "3.9" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/E3SM-Project/zppy-interfaces/archive/v{{ version }}.tar.gz
+  sha256: c847ac77e15e1e2bc125e097f8551f98c0d2b12c25a834ea8b7d8787a34f10b3
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  noarch: python
+
+requirements:
+  # Build from https://github.com/E3SM-Project/zppy-interfaces/blob/v{{ version }}/conda/dev.yml
+  host:
+    - python {{ python_min }}
+    - pip
+    - setuptools >=60
+  # Base from https://github.com/E3SM-Project/zppy-interfaces/blob/v{{ version }}/conda/dev.yml
+  run:
+    - python >={{ python_min }}
+    - beautifulsoup4
+    - lxml
+    - matplotlib-base >=3.8.2,<3.10
+    - mpas_tools >=0.21.0
+    - netcdf4
+    - numpy >=2.0,<3.0
+    - output_viewer ==1.3.3
+    - xarray >=2023.02.0
+    - xcdat >=0.7.3,<1.0
+
+test:
+  requires:
+    - pip
+    - python {{ python_min }}
+  imports:
+    - zppy_interfaces
+  commands:
+    - zi-global-time-series --help
+    - pip check
+
+about:
+  home: https://github.com/E3SM-Project/zppy-interfaces
+  summary: Interfaces between external tools and zppy, E3SM post-processing toolchain
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - mahf708
+    - xylar
+    - tomvothecoder
+    - forsyth2
+    - chengzhuzhang
```

Comment:
@mahf708, @tomvothecoder, @forsyth2, @chengzhuzhang

Can you all confirm by replying to this message that you are willing to be maintainers of zppy-interfaces?

Reply:
I'm willing to. Thanks, @xylar

Reply:
Yes, I'm willing, thanks @xylar

Reply:
Yes I am willing, thanks @xylar.